src/api/certificateAuthority.js

```diff
@@ -1,7 +1,7 @@
 import Q from "q";
 import logger from "winston";
 import pem from "pem";
-import { Keystore, Certificate } from "../model/keystore";
+import { KeystoreModelAPI, CertificateAPI } from "../model/keystore";
```

Comment:
hmmmm i see... And having a look it it now i think the naming for that variable is more correct. Im wonder if we should rename these variables to be more description in terms of what they are, something like `KeystoreModelAPI`. This would indicate its a keystore model used for the API calls